### PR TITLE
Set up a 'Hello Microverse' project

### PR DESCRIPTION
**Pull request by brainskev**
Created a new GitHub repo.
Created a new branch according to the [GitHub flow]
Added a .gitignore file.
Set up [a linter for HTML and CSS]) and made sure that any files that are not relevant are ignored by git.
Added descriptive README file 
Added .html,.css, test.md files

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -3,7 +3,7 @@ body {
 }
 
 h1 {
-  color: blue;
+  color: rgb(26, 26, 230);
 }
```
Comment (Abenezer-Tilahun) [inline styles.css:7]: - [ ] As stated in the project requirement your .css file should includes one class with styles for your header (make it in your favorite color), Kindly update it.
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @brainskev  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional README 
✔️ Good commit messages

almost

## Required Changes ♻️

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (brainskev) [reply styles.css:7]: Thanks for the observation. I have now made the changes

**Review by christianbiring1 — APPROVED** [review]
Hi @brainskev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
nicee

## Highlights
- [ ] All linters checked are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**